client/app/bundles/Events/reducers/AttendanceFormReducer.jsx

```diff
@@ -0,0 +1,38 @@
+import {
+  LOOK_UP_MEMBER, SET_ATTENDANCE_ATTRIBUTE, CREATE_ATTENDANCE
+} from '../actions/types';
+
+const INITIAL_STATE = {
+  'primary-email-address': '',
+  'given-name': '',
+  'family-name': '',
+  'primary-phone-number': '',
+  'disabled': ''
+};
+
+export default (state = INITIAL_STATE, action) => {
+  switch (action.type) {
+  case SET_ATTENDANCE_ATTRIBUTE:
+    const { prop, value } = action.payload;
+    return { ...state, [prop]: value };
```

Comment:
ES6 syntax. 
if props = 'email' and value = 'test@test.com' you'll get everything that's on state + email: 'test@test.com'